addon/spaniel-engines/ember-spaniel-engine.js

```diff
@@ -1,5 +1,5 @@
 import Ember from 'ember';
-const rAF = (typeof window === 'object') && typeof window.requestAnimationFrame === 'function' ? window.requestAnimationFrame : () => {};
+const rAF = (typeof window === 'object') && typeof window.requestAnimationFrame === 'function' ? window.requestAnimationFrame : (callback) => callback();
```

Comment:
So we're addressing this by immediately invoking rather than just closing over and no-oping, correct? How can we test this to ensure we're not leaking? Can we do something like wiring up [memwatch](https://github.com/lloyd/node-memwatch) to determine if we're leaking, and write a test that tries to force a leak?